Rakefile

```diff
@@ -5,7 +5,44 @@ require "rspec/core/rake_task"
 
 RSpec::Core::RakeTask.new(:spec)
 
-task default: :spec
+def google_auth_client(additional_parameters = nil)
+  require "google/api_client/client_secrets"
+
+  client_secrets = Google::APIClient::ClientSecrets.load("client_secrets.json")
```

Comment:
out of curiosity do we have a file of client_secrets.json?

Reply:
I've changed this to check for the file beforehand. The file is not included in the repo since it contains secrets. It can be downloaded from the Google Developer Console.